### DISCUSSION TITLE
Cannot load module `unist-util-visit`

### DISCUSSION THREAD

**Discussion by nick-cjyx9** [ElMassimo/iles#275] [Q&A] [answered]
I'm making a remark plugin for iles which uses the module `unist-util-visit`. I import it through `import { visit } from 'unist-util-visit'`, but error log shows that iles imported the module by `require`.

min reproduction: https://github.com/nick-cjyx9/iles-reproduction-fail-to-load

```
X [ERROR] "unist-util-visit" resolved to an ESM file. ESM file cannot be loaded by `require`. See http://vitejs.dev/guide/troubleshooting.html#this-package-is-esm-only for more details. [plugin externalize-deps] 
````

Thanks

**Comment by TechAkayy** [comment, marked as answer]
Adding `"type": "module"` to your `package.json` might be the easiest fix.
